src/components/WalletModal/index.tsx

```diff
@@ -314,31 +309,6 @@ export default function WalletModal({
       )
     }
 
-    function getTermsOfService(nftFlagEnabled: boolean, walletView: string) {
-      if (nftFlagEnabled && walletView === WALLET_VIEWS.PENDING) return null
-
-      const content = (
-        <Trans>
-          By connecting a wallet, you agree to Uniswap Labs’{' '}
-          <ExternalLink href="https://uniswap.org/terms-of-service/">Terms of Service</ExternalLink> and consent to its{' '}
-          <ExternalLink href="https://uniswap.org/privacy-policy">Privacy Policy</ExternalLink>.
-        </Trans>
-      )
-      return nftFlagEnabled ? (
-        <AutoRow style={{ flexWrap: 'nowrap', padding: '4px 16px' }}>
-          <ThemedText.BodySecondary fontSize={16} lineHeight="24px">
-            {content}
-          </ThemedText.BodySecondary>
-        </AutoRow>
```

Comment:
I would appreciate @aballerr input here, but it seems like this section is still to be included

Reply:
I tracked down the original pr, it looks like Charlie is right and this should stay.  

https://github.com/Uniswap/interface/pull/4790

Reply:
will add back